projects/libwebsockets/project.yaml

```diff
@@ -3,7 +3,6 @@ language: c
 primary_contact: "andy@warmcat.com"
 sanitizers:
   - address
-  - memory:
```

Comment:
You don't want MSAN and UBSAN?

Reply:
I disabled to avoid build fail.

Reply:
probably better to fix if possible (UBSAN will probably be a lot easier to fix than MSAN which will require compiling everything linked against your binary with MSAN)